src/commands/commerce/init.js

```diff
@@ -68,7 +67,7 @@ export class InitCommand extends Command {
       } else {
         await createRepo(githubOrg, githubRepo, templateOrg, templateRepo)
         openBrowser('https://github.com/apps/aem-code-sync/installations/select_target')
-        const res = await promptConfirm('Did you install the AEM Code Sync bot?')
+        const res = await promptConfirm('Did you install the AEM Code Sync bot by selecting your repository and clicking "save"?')
```

Comment:
Same here, not a big deal but did you get the language vetted by product or someone from docs?

Reply:
@jeff-matthews